holoviews/core/spaces.py

```diff
@@ -1221,6 +1220,31 @@ def dynamic_relabel(obj, **dynkwargs):
         return relabelled
 
 
+    def split_overlays(self):
+        """
+        Splits a DynamicMap into its components. Only well defined for
+        DynamicMap with consistent number and order of layers.
+        """
+        if not len(self):
+            raise ValueError('Cannot split DynamicMap before it has been initialized')
+        elif not issubclass(self.type, CompositeOverlay):
+            return [None], self
```

Comment:
``[None]`` is a little weird as a list of keys and I don't think you could get this from splitting a dynamic map.

Reply:
True, it just returns ``None`` in that case.